### PR TITLE
Add types

### PR DESCRIPTION
**Pull request by ChristianIvicevic**
I'd appreciate if you check the function signatures for `options.content` and `options.group` for correctness.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +33,10 @@
     "unist-util-visit": "^2.0.0"
   },
   "devDependencies": {
+    "@types/hast": "^2.3.1",
```
Comment (ChristianMurphy) [inline package.json:36]: unless there is a specific hast patch/minor version being relied on, it's good to use a wider semver range.
```suggestion
    "@types/hast": "^2.0.0",
```

File: package.json [hunk 2]
```diff
@@ -32,8 +33,10 @@
...
     "bail": "^1.0.0",
     "browserify": "^16.0.0",
+    "dtslint": "^4.0.5",
```
Comment (ChristianMurphy) [inline package.json:39]:
```suggestion
    "dtslint": "^4.0.0",
```

File: package.json [hunk 3]
```diff
@@ -44,6 +47,7 @@
     "tape": "^5.0.0",
     "tinyify": "^2.0.0",
     "to-vfile": "^6.0.0",
+    "unified": "^9.2.0",
```
Comment (ChristianMurphy) [inline package.json:50]:
```suggestion
    "unified": "^9.0.0",
```

**Review by ChristianMurphy — APPROVED** [review]
thanks @ChristianIvicevic!

**Review by ChristianMurphy — APPROVED** [review]

**Review by remcohaszing — APPROVED** [review]

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
thanks @ChristianIvicevic, released!

**Pull request merged**